instances/latest/terminologies/analysisTechnique/eyeMovementAnalysis.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/analysisTechnique/eyeMovementAnalysis",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/AnalysisTechnique",
+  "definition": "'Eye movement analysis' refers to a group of techniques used to measure eye movements from video or images.",
```

Comment:
is "measure" the right word here? Shouldn't it be "...group of techniques used to analyze eye movements..."?